app/models/host/base.rb

```diff
@@ -87,8 +87,8 @@ def dup
              :subnet, :subnet_id, :subnet_name,
              :subnet6, :subnet6_id, :subnet6_name,
              :domain, :domain_id, :domain_name,
-             :hostname, :fqdn, :fqdn_changed?,
-             :fqdn_was, :shortname,
+             :hostname, :fqdn, :saved_change_to_fqdn?,
+             :fqdn_before_last_save, :shortname,
```

Comment:
Do we actually use these anywhere? i think it may be leftovers from something, i couldn't find anywhere where these methods are called on a host

Reply:
removed these.